src/vs/workbench/services/auxiliaryWindow/browser/auxiliaryWindowService.ts

```diff
@@ -36,7 +36,8 @@ export interface IAuxiliaryWindowOpenEvent {
 export enum AuxiliaryWindowMode {
 	Maximized,
 	Normal,
-	Fullscreen
+	Fullscreen,
+	Custom
```

Comment:
@justschen can we have a follow up issue to clean this up, this seems unclear for the reader what it does. What does "Custom" mean? If the intent is to show a title bar, then lets make this explicit and not through the window mode.

Reply:
I opened https://github.com/microsoft/vscode/issues/224051